backend/sync-mounter-service/src/jvmMain/kotlin/Main.kt

```diff
@@ -0,0 +1,26 @@
+package dk.sdu.cloud.sync.mounter
+
+import dk.sdu.cloud.auth.api.AuthenticatorFeature
+import dk.sdu.cloud.micro.*
+import dk.sdu.cloud.service.CommonServer
+import dk.sdu.cloud.sync.mounter.api.SyncMounterServiceDescription
+
+data class SyncMounterConfiguration(
+    val cephfsBaseMount: String? = null,
+    val syncBaseMount: String? = null,
```

Comment:
Use defaults which match the hard-coded values you have all over the code. e.g. `/mnt/sync` should be the default value of `syncBaseMount`.